### ISSUE TITLE
grab listfile from releases instead

### ISSUE THREAD

**Issue by ferronn-dev** [lua-wow-tools/luadbd#5]
See https://github.com/wowdev/wow-listfile/blob/839cc13b286e79941ef8a430a0fda7e2d40092c9/README.md.

**Comment by Vladinator** [comment]
Does the issue boil down to this line requiring an update?

https://github.com/lua-wow-tools/luadbd/blob/main/luadbd/dbds.lua#L20

For example to replace it with this one instead:

```lua
local listfile = fetchHttp('https://github.com/wowdev/wow-listfile/releases/latest/download/community-listfile.csv')
```

Is this good enough of a fix for this issue?

**Comment by ferronn-dev** [comment]
The fix is present in luadbd-0.7.
